tests/mocha/field_checkbox_test.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2019 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+suite ('Checkbox Fields', function() {
+  function assertValue(checkboxField, expectedValue, expectedText) {
+    var actualValue = checkboxField.getValue();
+    var actualText = checkboxField.getText();
+    assertEquals(actualValue, expectedValue);
+    assertEquals(actualText, expectedText);
```

Comment:
+1 for checking both the value and the text. That'll make supporting screen readers easier.